test/controllers/EditSetControllerSpec.scala

```diff
@@ -48,868 +52,81 @@ class EditSetControllerSpec extends BaseSpec {
   "EditSetController GET /edit-set/{id}" should {
 
     "render the edit set page from a new instance of controller" in {
-      val messages: Map[String, Map[String, String]] =
-        Map(
-          "en" -> Map(
-            "edit-set.heading"       -> "Edit set: COAL 80 Sample",
-            "edit-set.table-caption" -> "Showing {0} - {1} of {2} records"
-          )
-        )
-      val mockMessagesApi = stubMessagesApi(messages)
-      val editSetInstance = inject[editSet]
-      val stub = stubControllerComponents()
+      val mockEditSet = mock[editSet]
```

Comment:
I would have expected constructing a real instance of the view; like this:

```
        new editSet(any[User], anyString(), anyString(), any[EditSetPage], any[EditSetEntryRowOrder])(
          any[Messages],
          any[Request[AnyContent]]
        )
```

@rwalpole, I believe you mentioned this, this morning?

Reply:
@tna-erasmos I don't know if this would work exactly as the editSet view constructor expects GovukPagination and GovukTable components as well. A mock is fine here and we will worry about constructing a view in the EditSetViewSpec.

Reply:
OK.